pkg/agent/bakerapi_test.go

```diff
@@ -462,65 +411,5 @@ var _ = Describe("AgentBaker API implementation tests", func() {
 				Expect(config.Gallery).To(Equal("aksazurelinux"))
 			}
 		})
-
-		It("should return correct value for all existing distros with linux node image version override", func() {
```

Comment:
just want to confirm we're not losing any coverage by removing this

Reply:
I don't think we are. However, I've added it back - figured out a way to simulate the old behavior. Better safe than sorry.